extensions/amp-story-auto-ads/0.1/story-ad-localization.js

```diff
@@ -18,7 +18,7 @@ import {
   LocalizedStringId,
   createPseudoLocale,
 } from '../../../src/localized-strings';
-import {Services} from '../../../src/services';
+import {getLocalizationService} from '../../amp-story/1.0/amp-story';
```

Comment:
If possible: we should not bundle some `amp-story` code directly into the `amp-story-auto-ads`, can you replicate what we do for the `storeService` and use an async service getter here?

https://github.com/ampproject/amphtml/blob/cec4cce04108ac952f00478f300e9c4f005c8319/extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js#L142-L147

Reply:
This would require changing code in auto-ads to use the service asynchronously, I just replicated the getter in auto-ads.js so that we don't have to bundle the code.